lib/game.rb

```diff
@@ -0,0 +1,94 @@
+require 'pry'
+class Game
+  attr_accessor :board, :player_1, :player_2
+  WIN_COMBINATIONS = [[0,1,2],[3,4,5],[6,7,8],[0,3,6],[1,4,7],[2,5,8],[0,4,8],[2,4,6]]
+
+  def initialize(player_1 = Players::Human.new("X"), player_2 = Players::Human.new("O"), board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def current_player
+    if board.turn_count.even?
+      player_1
+    else
+      player_2
+    end
+  end
+
+  def over?
+    won? || board.full?
+  end
+
+  def won?
+    WIN_COMBINATIONS.each do |combo|
+      board_combo = []
+      combo.each do |index|
+        board_combo << board.cells[index]
+      end
+      if board_combo.uniq.count <= 1
+        if board_combo.uniq[0] == "X" or board_combo.uniq[0] == "O" #if you ever run into a winning combo, break out of loop
+          return board_combo.uniq
+        end
+      end
+    end
+    return false # if you make it through the entire loop with no luck, it's false
+  end
+
+  def draw?
+    !won? && over?
+  end
+
+  def winner
```

Comment:
Instead of calling `Game#won?` twice, consider using an if statement assignment. For example:

```ruby
if won = won?
  board.cells[won.first]
end
```

If `won?` returns true, then and only then will the local variable `won` be assigned, otherwise, the method will just abort with a false return value. But if `won` is assigned, we can use that variable instead of calling `won?` again within the if statement.